unit-tests/run-unit-tests.py

```diff
@@ -365,6 +377,8 @@ def get_tests():
 
     # Python unit-test scripts are in the same directory as us... we want to consider running them
     # (we may not if they're live and we have no pyrealsense2.pyd):
+    if test_dir:
```

Comment:
Please don't change the global variable current_dir, it's used as the script directory also. Use a local variable to assign test_dir or current_dir to search for the tests

Reply:
Done. Point taken and incorporated these comments.